server/routes/termdb.config.ts

```diff
@@ -206,13 +206,21 @@ function addGenomicQueries(c, ds, genome) {
 		}
 	}
 
+	if (q.DZImages) {
+		q2.DZImages = {}
+		for (const k in q.DZImages) {
+			q2.DZImages[k] = JSON.parse(JSON.stringify(q.DZImages[k]))
```

Comment:
should only reveal `type` property but not folder path

Reply:
Fixed.